.agents/skills/superposition/reference/documentation/other/client_experimentation.md

```
@@ -0,0 +1,298 @@
+
+```
+
+### Experiment Client Methods Reference
+
+#### Run polling for updates from Superposition Service
+
+the Experimentation client polls for updates from the superposition service and loads any changes done on the server. This means that experiments changed in superposition are reflected on the client in the duration of `polling_interval`. `run_polling_updates()` should be run in a separate thread, as it does not terminate.
+
+##### Function definition
+
+ ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)